distribution/docs/src/main/resources/content/_metadataReference/core-attributes-table.adoc

```diff
@@ -138,16 +138,16 @@ a|The modification date of the resource http://dublincore.org/documents/dcmi-ter
 |Alpha-3 language code(s) per ISO_639-2
 |
 
-|[[_resource.derived-download-url]]resource.derived-download-url
-|Download URL(s) for accessing the derived formats for the metacard resource.
+|[[_resource.derived-uri]]resource.derived-uri
+|Catalog-specific Location(s) for accessing the derived resources for a metacard resource. This URI is used for internal catalog requests.
```

Comment:
Maybe there is a way to not have the word "resource" appear twice in the sentence. How about 

> ...for accessing the resources derived from another source (e.g. an overlay of a larger image)

Reply:
updated